tests/conftest.py

```diff
@@ -58,6 +59,28 @@ def default_prediction_server_id():
     return "5f06612df1740600260aca72"
 
 
+@pytest.fixture()
+def serverless_prediction_environment():
+    short_uuid = str(uuid.uuid4())[:5]
+    import datarobot as dr
+
+    prediction_environment = dr.PredictionEnvironment.create(
```

Comment:
should use the cleanup context manager here or alternatively  just do a  since this step need not be done with idp helpers

yield prediction_environment.id
prediction_environment.delete()